models/product-type/src/product-type/product-type-draft/presets/sample-data-goodstore/furniture-and-decor.ts

```diff
@@ -0,0 +1,299 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  AttributeDefinitionDraft,
+  AttributeBooleanTypeDraft,
+  AttributeLocalizedEnumTypeDraft,
+  AttributeLocalizableTextType,
+  AttributeLocalizedEnumValueDraft,
+} from '../../../../index';
+import type { TProductTypeDraftBuilder } from '../../../types';
+import * as ProductTypeDraft from '../../index';
+
+const furnitureAndDecor = (): TProductTypeDraftBuilder =>
+  ProductTypeDraft.presets
+    .empty()
+    .key('furniture-and-decor')
+    .name('Furniture and decor')
+    .description(
+      'items with color, fabric, size, finish and/or product spec attributes'
+    )
+    .attributes([
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('productspec')
+        .type(AttributeLocalizableTextType.random())
+        .label(
+          LocalizedString.presets
+            .empty()
+            ['en-GB']('Product Specifications')
+            ['de-DE']('Produktspezifikationen')
+        )
+        .inputTip(
+          LocalizedString.presets
+            .empty()
+            ['en-GB'](
+              'Details on the specifications of the product, including size, wood type, fabric type, to be detailed in bullet points.'
+            )
+        )
+        .isRequired(false)
+        .attributeConstraint('SameForAll')
+        .isSearchable(false)
+        .inputHint('MultiLine'),
```

Comment:
All of the attribute definitions in this product type have a `displayGroup` [attribute](https://github.com/frontastic-developers/customer-thegoodstore/blob/59b614b30c58410dff5d23bb9e4e6548566a2d4f/getting-started/data/product-type.json#L27)

Reply:
Is that a supported field? I don't see it documented